src/libs/actions/Report.js

```diff
@@ -1017,8 +995,17 @@ function deleteReportComment(reportID, reportAction) {
         ],
     };
 
-    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`, reportActionsToMerge).then(() => {
-        setLocalLastRead(reportID, getLastReadSequenceNumber(reportID));
+    // If the comment we are deleting is more recent than our last read comment we will update the unread count
```

Comment:
To clarify, this would happen if a message below the New banner was to be deleted, correct? 

What other purpose does the sequence number have other than placing the New banner in the correct spot?

Reply:
>To clarify, this would happen if a message below the New banner was to be deleted, correct?

Yep!

>What other purpose does the sequence number have other than placing the New banner in the correct spot?

This is a fairly broad question and could take a while to answer completely. 

Generally speaking they are used for...

- ordering report comments
- figuring out the most recent report action (highest sequenceNumber)
- pagination (offset is typically the oldest locally stored sequenceNumber)
- calculating whether a chat is read / unread and how many unread messages a chat has
- setting the new marker position
- etc...

If you have more specific questions I would be glad to answer them.